README.md

```diff
@@ -1,2 +1,211 @@
-# testpilot-metrics
-JS library that handles metrics events for Test Pilot experiments
+# `testpilot-metrics`
+
+The `testpilot-metrics` library sends pings to Google Analytics and Mozilla's
+internal metrics pipeline. It is designed for use by Test Pilot experiments.
+
+
+## Installation
+
+`npm install testpilot-metrics`
+
+The only file you need in your build chain is `testpilot-metrics.js`. It has no
+dependencies.
+
+
+## Usage
+
+### Quick start
+
+Here's a simple node-style example:
+
+```js
+
+const Metrics = require('testpilot-metrics');
+
+const { sendEvent } = new Metrics({
+  id: '@my-addon',
+  cid: 'some-non-PII-user-ID',
```

Comment:
:+1: Thanks for the suggestion. I've gone back and forth on this point; not having node-style module support in WebExtensions makes it really, really hard to find an off-the-shelf library that's easy to use. But I guess I could roll my own, using `window.crypto` to get pseudorandom data, and resort to the hidden window trick to get the `crypto` API for SDK/bootstrapped add-ons. I'll add that now